pyro_risks/evaluate.py

```diff
@@ -157,14 +158,7 @@ def predict(x):
 
 
 def main(args):
-    usecols = [cfg.DATE_VAR, cfg.ZONE_VAR, cfg.TARGET] + cfg.PIPELINE_ERA5T_VARS
-    pipeline_vars = [cfg.DATE_VAR, cfg.ZONE_VAR] + cfg.PIPELINE_ERA5T_VARS
-    df = pd.read_csv(cfg.ERA5T_VIIRS_PIPELINE, usecols=usecols)
-    df["day"] = df["day"].apply(
-        lambda x: datetime.strptime(str(x), "%Y-%m-%d") if not pd.isnull(x) else x
-    )
-    X = df[pipeline_vars]
-    y = df[cfg.TARGET]
+    X, y = load_dataset()
```

Comment:
The refactoring is nice, but I'm not sure about the point here?
You removed the script part but kept "def main...". As you suggested in the previous comments, either we keep those and accept scripts inside the lib, or we move everything outside to another folder. Either way, I wouldn't recommend keeping `def main` if we aren't turning this into a script :sweat_smile:

Reply:
Ok, I had an amazing idea concerning the training prediction and orchestration and I got way to excited 😄
I wanted to explore it really quickly and also not do a large PR (but small ones) which sounds reasonable. 
However, I totally should have opened an other issue for discussing the design first!

 I will do that and I am pretty sure my code will be cleaner and make more sense and the review will be shorter for you 😆